home/start_page.py

```diff
@@ -98,18 +97,28 @@ def read_config(self):
 
     def render(self):
         """Rendering all apps."""
-        self.output.clear_output()
-        apps = self.load_apps()
+        home = self._create_home_widget()
+        display(home)
+
+        from pathlib import Path
+
+        config_dir = Path("../../.aiidalab").resolve().absolute()
```

Comment:
This feels more robust and simple?

```suggestion
        config_dir = Path.home()  / ".aiidalab"
```

Reply:
Also this should probably be a helper function somewhere outside of this notebook. Is this directory used anywhere else in home app?

Reply:
Nice! Didn't know about `.home()`. 100% more robust/simple.

As for the `.aiidalab` dir being used elsewhere, maybe not at the moment, but I already have some ideas for other uses. So yes, good to make it easily accessible by some means.

Reply:
Can't think of where to place it. Let's discuss maybe for a separate PR.